accounts/abi/bind/auth.go

```diff
@@ -45,15 +45,15 @@ func NewTransactor(keyin io.Reader, passphrase string) (*TransactOpts, error) {
 }
 
 // NewKeyStoreTransactor is a utility method to easily create a transaction signer from
-// an decrypted key from a keystore
-func NewKeyStoreTransactor(keystore *keystore.KeyStore, account accounts.Account) (*TransactOpts, error) {
+// a decrypted key from a keystore.
+func NewKeyStoreTransactor(keyStore *keystore.KeyStore, account accounts.Account) (*TransactOpts, error) {
```

Comment:
keystore is a single word, please don't camel case it

Reply:
On it